### ISSUE TITLE
NVDA doesn't speak the diacritics of some IPA symbols in applications other than MS Word

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5011]
**Reported by Sukil on 2015-03-31 18:14**
Steps to reproduce:
1. Type for example the letter "s" in MS Word, followed by the symbol 0x33A. Both symbols should merge and eSpeak should only play the sound associated with the symbol when reading character by character.
2. Copy that symbol into some other application (tested with EspeakEdit and PowerPoint). When reading character by character, NVDA ignores the diacritics and says "s". In addition, when pressing NVDA+Numpad5 two and three tymes, the description of a regular "s" is given.

**Comment by bhavyashah** [comment]
Could someone please test this and revert about the current standing of this issue? If no one can confirm its presence within about a week or two, I would kindly suggest closing.

**Comment by Adriani90** [comment]
@Sukil are you still having this issue?

**Comment by enessaribas** [comment]
This is still an existing issue. IPA symbols are not read correctly by NVDA. This can be solved by allowing NVDA to read them with their unicode assigned names, found in the following issue.
https://github.com/nvaccess/nvda/issues/2922

**Comment by CyrilleB79** [comment]
With NVDA 2024.1beta5, I get the following:
* navigating character by character, either with caret (left/right arrows) or review cursor (numpad1/3) reports the "s" and the diacritic as a single character. It does not matter if I am in Word, PowerPoint or Notepad. To me, that's OK because the diacritic is a combining character.
* The same way, reporting the character with numpad2 reports only one character, namely, the sound /s/.
* A double press on the character reports "sierra  symbol 33A" with eSpeak; "symbol 33A" is the way eSpeak reports this diacritic alone. So it's OK to me.
* Three presses on the character report the combined character, i.e. the sound /s/. To me, it should report instead the two hex codes of "s" and of the diacritic, i.e. "83 0x53, 826 0x33a". That's a bit verbose, but that's the only way to report the same information; instead reporting again the compound character is useless.

It's worth noting that no one has commented on this issue after the beginning of 2019, when NVDA was still running Python 2.

@sukiletxe and others, do you confirm that only the triple press should be fixed? If yes, I'll take care of it.

**Comment by sukiletxe** [comment]
For me yes, that would be enough.
